sql/telemetry_derived/clients_daily_active_hours_by_top_city_aggregates_v1/query.sql

```diff
@@ -0,0 +1,65 @@
+WITH top_cities_t AS (
+  SELECT
+    COUNT(client_id) AS dau,
+    CONCAT(country, ":", geo_subdivision1, ":", geo_subdivision2, ":", city) AS geo
+  FROM
+    `moz-fx-data-shared-prod.telemetry.clients_daily`
+  WHERE
+    submission_date = "2020-03-01"
```

Comment:
Just flagging that this does need to be some fixed date to keep the set of cities constant.  No action needed though, unless you want to add a comment.